File: samples/snippets/csharp/System/string.split/compiler-resolution.cs
using System;

namespace Split
{
    class CompilerResolution
    {
        public static void Main6()
        {
            // <Snippet1>
            string value = "This is a short string.";
            Char delimiter = 's';
            string[] substrings = value.Split(delimiter);
            foreach (var substring in substrings)
                Console.WriteLine(substring);

            // The example displays the following output:
            //     Thi
            //      i
            //      a
            //     hort
            //     tring.
            // </Snippet1>
        }
    }
}

File: snippets/fsharp/System/String/Split/compiler-resolution.fs
module compiler_resolution
// <Snippet1>
let value = "This is a short string."
let delimiter = 's'
let substrings = value.Split delimiter
for substring in substrings do
    printfn $"{substring}"

// The example displays the following output:
//     Thi
//      i
//      a
//     hort
//     tring.
// </Snippet1>